### PR TITLE
Feature/984 article upload errors

### PR DESCRIPTION
**Pull request by richard-jones**

### CODE REVIEW

File: portality/templates/doaj/publishers.html [hunk 1]
```diff
@@ -454,41 +454,160 @@
...
+to DOAJ.  Depending on the message in the "Notes" column of your <a href="{{ url_for('publisher.upload_file') }}">uploads</a> table,
+some corrective actions may be available to you.</p>
+
+<p>In cases where there is no action to be taken, or the suggested actions have not successfully resolved the issues,
+please <a href="{{ url_for('doaj.contact') }}">contact us</a> with the following details:</p>
+
+<ul>
+    <li>The time of the file upload</li>
+    <li>The exact error message as it appears in the "Notes" column</li>
+    <li>Your publisher account name</li>
```
Comment (emanuil-tolev) [inline portality/templates/doaj/publishers.html]: username, not name

**Comment by richard-jones** [comment]
Updated the text "name" -> "username"

File: portality/templates/publisher/uploadmetadata.html [hunk 2]
```diff
@@ -104,6 +104,7 @@
                     <td>
                         {% if file.error %}
                             {{file.error}}
+                            <a href="{{ url_for('doaj.publishers', _anchor=file.error) }}">(explain)</a>
```
Comment (emanuil-tolev) [inline portality/templates/publisher/uploadmetadata.html]: Ahaaaa, that's why the anchors have full sentences with spaces for names ... very clever.

**Comment by emanuil-tolev** [comment]
Perfect, thanks @richard-jones . Release asap I suppose?

**Comment by emanuil-tolev** [comment]
> Release asap I suppose?

As in, do you want to show it to DOAJ team on test first, wait for some feedback, make more changes etc.? Or just make it live.

**Pull request merged**